Tests/NumLib/TestComponentNorms.cpp

```diff
@@ -143,7 +143,7 @@ TEST(MPITest_NumLib, ComponentNormMultiComponent1)
     unsigned const num_components = 3;
     auto const norm_type = MathLib::VecNormType::NORM1;
     auto const tolerance =
-        num_components * 600 * std::numeric_limits<double>::epsilon();
```

Comment:
This is strange. Without vectorization I'd assume that the situation becomes better. But the resulting tolerance is still OK.